src/scale-manager/README.md

```diff
@@ -1,3 +1,22 @@
 # Scale Manager
 
-Simplifies listening to resize events by passing the relevant data to a provided callback.
\ No newline at end of file
+A utility class for listen for resize events.
+
+```javascript
+var scaleManager = new ScaleManager(function(resizeData) {
+  console.log('This is called on window resize');
+
+  console.log('Window width is', resizeData.width);
+  console.log('Window height is', resizeData.height);
+  console.log('Window aspect ratio is', resizeData.ratio);
+});
+```
+
+If you do not have a listener ready at the time of construction or you need to replace the existing listener, the
+`enable` method exists for that purpose:
```

Comment:
The
`enable` method exists in case you do not have a listener ready at the time of construction or need to replace the existing listener.